packages/create-amplify/src/npm_project_initializer.ts

```diff
@@ -24,15 +26,27 @@ export class NpmProjectInitializer {
       // if package.json already exists, no need to do anything
       return;
     }
-    this.logger.log(
+    await this.logger.debug(
       'No package.json file found in the current directory. Running `npm init`...'
     );
 
     try {
-      await this.execa('npm', ['init', '--yes'], {
-        stdio: 'inherit',
+      let aggregatedStdout = '';
+      const aggregatorStream = new stream.Writable();
+      aggregatorStream._write = function (chunk, encoding, done) {
+        aggregatedStdout += chunk;
+        done();
+      };
+
+      const childProcess = this.execa('npm', ['init', '--yes'], {
+        stdin: 'inherit',
         cwd: this.projectRoot,
       });
+
+      childProcess?.stdout?.pipe(aggregatorStream);
```

Comment:
what about `stderr`?

Reply:
I'll add `stderr`.

Reply:
Done in [52a5eda](https://github.com/aws-amplify/amplify-backend/pull/595/commits/52a5eda31ca1d6dc5bfdca59e500c1ed746c0669)